src/generated/resources/pack.mcmeta

```diff
@@ -1,14 +1,14 @@
 {
   "pack": {
     "description": {
-      "translate": "pack.forge.description"
+      "translate": "pack.neoforge.description"
     },
     "neoforge": {
       "versions": {
         "client_resources": 22,
```

Comment:
Is 22 the correct client resources version for 1.20.4?

Reply:
yep 
https://minecraft.wiki/w/Pack_format